string desired_image_path
string init_image_path
---
float64 lambda_g